src/views/components/input/file/InputFile.native.js

```diff
@@ -1,21 +1,52 @@
 import React, { Component } from 'react';
-import { Box, Text } from '../../../components';
+import Expo, { ImagePicker } from 'expo';
+import { View, Image, Button } from 'react-native';
 
 class InputFile extends Component {
   static defaultProps = {
   }
 
   static propTypes = {
   }
+  
+  state = {
+    image: null,
+  };
+
+  handlePress = async () => {
+    Expo.Permissions.askAsync( Expo.Permissions.CAMERA_ROLL ).then(
```

Comment:
Can we convert to using async/await instead of the `.then()` syntax since the `handlePress` function is already prefixed with `async`? (please also use `try`/`catch`)